api/base/views.py

```diff
@@ -455,6 +456,33 @@ def error_404(request, format=None, *args, **kwargs):
     )
 
 
+class BaseChildrenList(JSONAPIBaseView):
+    permission_classes = (
+        ContributorOrPublic,
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        ReadOnlyIfRegistration,
+        base_permissions.TokenHasScope,
+        ExcludeWithdrawals,
+    )
+    ordering = ('-modified',)
+
+    def get_default_queryset(self):
+        return default_node_list_queryset(model_cls=self.model_class)
+
+    # overrides ListAPIView
```

Comment:
```suggestion
    # overrides GenericAPIView
```
I was confused looking for how `ListAPIView` was used